### ISSUE TITLE
Can't achieve the given performance: ResNet-50 + SSP+CE(Uniform) for imageNet-LT

### ISSUE THREAD

**Issue by ChCh1999** [YyzHarry/imbalanced-semi-self#17]
I download the pre-trained model from the given path [Resnet-50-rot](https://drive.google.com/file/d/1XritMl3dYa9iW-TomaKU1XLQJVqgopMz/view?usp=sharing). And train the model with the given config `imagenet_inat/config/ImageNet_LT/feat_uniform.yaml`
The training cmd is:
`python imb_cls/imagenet_inat/main.py --cfg 'imb_cls/imagenet_inat/config/ImageNet_LT/feat_uniform.yaml' --model_dir workdir/pretrain/moco_ckpt_0200.pth.tar`. 
I only get `41.1` top-1 accuracy but the given model achieved 45.6 [CE(Uniform) + SSP].

Can you help me check where is the problem？
![image](https://user-images.githubusercontent.com/34148951/119098782-caeeee80-ba48-11eb-889a-d8d2d6e6f798.png)
![image](https://user-images.githubusercontent.com/34148951/119098815-d4785680-ba48-11eb-83a5-dd94a24265f4.png)

**Comment by YyzHarry** [comment]
Hi, I double checked the config, which seems fine to me. So currently I'm not sure what exactly causes the difference.

I would suggest that you could first run the baseline model (without SSP), and compare the model performance w/ & w/o SSP. If that gives you reasonable gains, then you might want to check whether the baseline model matches the number reported in our [paper](https://arxiv.org/pdf/2006.07529.pdf). If the performance of the baseline model is also lower than the number reported, then it might be due to the hyperparameters and the training settings.

Otherwise, you may want to check whether the pre-trained weights are loaded correctly, or the exact training setting such as PyTorch version (1.4 for this repo), etc.

**Comment by ChCh1999** [comment]
The baseline result is 
![image](https://user-images.githubusercontent.com/34148951/119160060-75d3cc80-ba8a-11eb-9e05-1d62641ce0e9.png)
my running environment is 
![image](https://user-images.githubusercontent.com/34148951/119160403-d19e5580-ba8a-11eb-889f-dab420ff78e7.png)
And I think the model is loaded correctly because I've tested the given model provided by you and got the correct result. 
Does the following output mean the weight for resnet50 loaded?
![image](https://user-images.githubusercontent.com/34148951/119160593-04e0e480-ba8b-11eb-93d8-409181a7a37a.png)
Thanks for your answer.

**Comment by YyzHarry** [comment]
Yes, I think that indicates your loading process is correct. Seems there's a reasonable gap though, but I'm not sure what causes the difference. I would suggest that you could tune the hyper-parameters a bit; I took a quick pass on the original [OLTR config](https://github.com/zhmiao/OpenLongTailRecognition-OLTR/blob/master/config/ImageNet_LT/stage_1.py#L21), and seems they use 0.1 as initial LR. Also, it is observed that training for longer epochs could lead to better results, so you might want to increase the epoch numbers to see what you can get. Hope this helps!
